code/datums/martial/cqc.dm

```diff
@@ -10,8 +10,7 @@
 	block_chance = 75
 	var/just_a_cook = FALSE
 	var/static/list/areas_under_siege = typecacheof(list(/area/crew_quarters/kitchen,
-														/area/crew_quarters/cafeteria,
-														/area/crew_quarters/bar))
+														/area/crew_quarters/kitchen/backroom))
```

Comment:
this is a typecache, `/area/crew_quarters/kitchen/backroom` would be included under `/area/crew_quarters/kitchen`